src/badge/Badge.tsx

```diff
@@ -9,7 +9,7 @@ import {
   ViewStyle,
 } from 'react-native';
 import { withTheme } from '../config';
-import { Theme } from '../config/theme';
+import { ThemeProps } from '../config/ThemeProvider';
```

Comment:
I think `ThemeProps` should be added to the `config/index.ts` and also exported in `src/index.ts` can you change that around and then simplify the imports so that they only have to import from `../config`?